lib/appsignal/utils.rb

```diff
@@ -49,8 +118,7 @@ def encode_utf8(value)
           )
         end
       end
-
-      extend ClassMethods
```

Comment:
was this fixed already? because you can undo this change. How it is now, changing it from a module to a singleton class, makes this a completely type of different object